### PR TITLE
Add snyk workflows

### PR DESCRIPTION
**Pull request by ymengesha**
https://hivemq.kanbanize.com/ctrl_board/41/cards/15245/details/

**Motivation**

**Changes**

### CODE REVIEW

File: .github/workflows/snyk-push.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - uses: hivemq/hivemq-checkout-composite-action@v1
+        with:
+          path: mqtt-cli
+
+      - name: Run Snyk monitor
+        uses: snyk/actions/gradle-jdk11@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+          ORG_GRADLE_PROJECT_hivemqCommonsUsername: ${{ secrets.PUBLISH_GITHUB_USERNAME }}
+          ORG_GRADLE_PROJECT_hivemqCommonsPassword: ${{ secrets.PUBLISH_GITHUB_TOKEN }}
```
Comment (SgtSilvio) [inline .github/workflows/snyk-push.yml:22]: Do we really need these credentials for the mqtt-cli?

File: .github/workflows/snyk-release.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      - uses: hivemq/hivemq-checkout-composite-action@v1
+        with:
+          path: mqtt-cli
+
+      - name: Run Snyk monitor
+        uses: snyk/actions/gradle-jdk11@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+          ORG_GRADLE_PROJECT_hivemqCommonsUsername: ${{ secrets.PUBLISH_GITHUB_USERNAME }}
+          ORG_GRADLE_PROJECT_hivemqCommonsPassword: ${{ secrets.PUBLISH_GITHUB_TOKEN }}
```
Comment (SgtSilvio) [inline .github/workflows/snyk-release.yml:22]: same here

File: .github/workflows/snyk-push.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  push:
+    branches:
+      - master**
+
+jobs:
+  run-snyk-monitor:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: hivemq/hivemq-checkout-composite-action@v1
```
[the remark this answers is not in the document]
Comment (ymengesha) [reply .github/workflows/snyk-push.yml]: I will update it.

File: .github/workflows/snyk-push.yml (hunk 1 above)
Comment (ymengesha) [reply .github/workflows/snyk-push.yml]: Good catch! We don't need them.

File: .github/workflows/snyk-release.yml (hunk 2 above)
Comment (ymengesha) [reply .github/workflows/snyk-release.yml]: We don't need them.

**Review by SgtSilvio — APPROVED** [review]

**Pull request merged**